control-plane/plugin/src/resources/node.rs

```diff
@@ -353,6 +369,17 @@ impl CreateRows for NodeDisplay {
                 }
                 rows
             }
+            NodeDisplayFormat::Labels => {
+                let mut rows = vec![];
+                for node in self.inner.iter() {
+                    let mut row = node.create_rows();
+                    let labelstring = NodeDisplay::get_label_list(node).join(", ");
+                    // Add the labels to each row.
+                    row[0].add_cell(Cell::new(&labelstring));
```

Comment:
I wasn't aware of this thing being done for cordon. Seems confusing that we add the header only if we have the label if the command is specifically get cordon or get labels
What do you think @Abhinandan-Purkait ?

Reply:
I too noticed this now, the ideal way would have been using a `--show-labels` on the `get nodes/node` but maybe fine for now since we are already doing it for cordon and drain? Not sure.

Reply:
kubectl get nodes has `--show-labels` for displaying the labels when the flag is passed. It adds a new column called labels to show the the labels comma separated. The normal get nodes doesn't have the labels column for obvious reasons i.e. it will overwhelm the get nodes output

Reply:
Maybe we can do the same here then?